Sources/XcodeGenKit/PBXProjGenerator.swift

```diff
@@ -538,6 +539,13 @@ public class PBXProjGenerator {
                     )
                     copyFrameworksReferences.append(embedFile.reference)
                 }
+                
+                let buildPath = Path(dependency.reference)
+                    .byRemovingBase(path: project.basePath)
+                    .components
```

Comment:
You should be able to use `parent.string` instead of `components.dropLast()`.joined(separator: "/")`